src/main/java/org/folio/clients/InventoryClient.java

```diff
@@ -42,6 +43,11 @@
     return getReferenceDataByUrl(endpoint, params, IDENTIFIER_TYPES);
   }
 
+  public Map<String, JsonObject> getLocations(OkapiConnectionParams params) {
+    String endpoint = resourcesPathWithPrefix(LOCATIONS) + "?limit=" + REFERENCE_DATA_LIMIT;
+    return getReferenceDataByUrl(endpoint, params, LOCATIONS);
+  }
```

Comment:
Also please update Integration tests in DataExportTest

Reply:
Updated